tests/launch_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+SANDBOX="../fssb -- "
+RUNNER="./tests.py"
+
+testcases=(
+	'test_no_syscalls'
+	'test_save_empty_file'
+)
+
+echo "Removing all /tmp/fssb-*"
+rm -rf -- /tmp/fssb-*
+
+echo "Launching tests"
+
+for testcase in ${testcases[@]}
+do
+	$SANDBOX $RUNNER test $testcase
```

Comment:
This didn't work for me. I needed to do `$SANDBOX python $RUNNER test $testcase`. Note the explicit `python`.

I think it's because there's an issue with forks in `fssb`. Right now we're not tracking forks. When you do a `./file.py`, the shell forks itself and executes the python with `file.py`. But when you explicitly give `python ./file.py`, there's no need to fork - just call `python`.

Very minor issue I think, but nevertheless :)